objects/ooptionicons/Step_0.gml

```diff
@@ -51,7 +51,7 @@ if (P1menuRight && menuCooldown < 1 && image_index = 1)
 		sfxVolume += 0.01;
 	}
 	
-		audio_group_set_gain(audiogroup_soundeffect, sfxVolume, 0);
+		audio_group_set_gain(audiogroup_sfx, sfxVolume, 0);
```

Comment:
Make sure the indentation is lined up with everything else in the if statement. It makes the code look more neat.